_capital/departments.md

```diff
@@ -42,87 +40,28 @@ For each of your departments, create a page using the following steps:
 
 ![The Templates settings area when adding or editing a page](../images/departments-page-templates-settings.png)
 
-## Add a navigation menu that lists all of your departments
 
-Now that you have department pages set up, you can create a navigation menu that lists your departments.
+### Add navigation menus for each department
 
-  1. Navigate to **Online Store ► Navigation** in your Shopify admin area.
-  2. Use the **Add menu** button to add a new menu.
-  3. Give your new menu a descriptive title, like `Departments`. <p>Depending on your shop's navigation set up, you may end up changing this title later on.</p>
-  4. Use the **Add menu item** button to add a new item. For example, let's add an item called "Mens".
-  5. Use the dropdown menu to change the Link field to **Page**.
-  6. Choose a page that correlates to a department page that  you made earlier.
-  7. Repeat steps 4–6 until you've added all of your departments to the menu.
+It's now time to associate each department page with your product collections. You can do this by creating a navigation menu for _each_ department that contains each of the department's sections. This menu tells the Department page which collections to display, and in what order. 
 
-This navigation menu will be used to display the list of departments in your store's navigation menu. The name of this menu will appear in your main menu, alongside your other main pages. You may want to name this menu `Shop` or `Departments`.
+For example, the Mens department now has its own department page. Within the Mens department, we want sections for boots, shoes, accessories, and any other mens-related products that we sell.
```

Comment:
I think we should throw some quotes around each instance of "Mens". There are two on this line, and a few below. Also switch the ones that are `Mens` to "Mens"—do you think that makes sense? Or should they all be `Mens`. My brain is a little fried right now... too laaate

Reply:
@cbmunro I can do that. I think "Men's" works!